ts/features/mvl/saga/networking/handleGetMvlDetails.ts

```diff
@@ -52,6 +59,9 @@ const convertMvlDetail = (
       },
       attachments: eml.attachments.map(convertMvlAttachment),
       metadata: {
+        id: msgId as MvlId,
+        timestamp: new Date(certData.data.timestamp),
```

Comment:
```suggestion
        timestamp: certData.data.timestamp,
```
If I have the last spec 🤣 
`export declare const UTCISODateFromString: t.Type<Date, string, unknown>;`
It is already a `Date`

Reply:
Yes you're right! I was looking to an old version of the specification and luckly was just an unneccessary operation but with the same meaning. Fixed in 2fa482b0da9865bae2dd2bce4c9339bdee56c515